book/src/installers/usage.md

```diff
@@ -0,0 +1,122 @@
+# Usage
+
+axo's installers are designed to require no end user input - however, in many
+cases, end users like the ability to tailor their installation experience. For
+these users, our installers allow several inputs which give the user more
+control over their installation experience via configuration. Most of these are
+implemented using environment variables, however several are also available as
+arguments to the installer scripts themselves.
+
+This documentation is targeted at developers using `dist` to distribute their apps. If you expect your users to be interested in any of these features, we recommend incorporating the relevant information into your own installation documentation.
```

Comment:
we might add a note here encouraging folks to file issues to request any further information that may help them document for their users